api/swagger/swaggerDoc.yaml

```diff
@@ -813,6 +1071,40 @@ definitions:
         type: boolean
         default: false
         example: true
+  AddressDataV03:
+    description: The data format of the input address
+    required:
+      - address
+    properties:
+      address:
+        $ref: '#/definitions/AddressModelV03'
+      isWorkAddress:
+        type: boolean
+        default: false
+        example: true
+      policyType:
+        type: string
+        default: simplye
+        example: simplye
+  AddressResponseV03:
+    type: object
+    properties:
+      status:
+        type: number
+        example: 200
+      type:
```

Comment:
And again--can this be renamed?

Reply:
It's confusing when reading the yaml, but for the API response I think it makes sense to have it there.